eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -86,8 +85,8 @@ protected RandomizationConfigResponse readCacheFile() throws IOException {
         return configResponse;
     }
 
-    public void setFlagsFromResponse(Reader response) {
-        RandomizationConfigResponse config = gson.fromJson(response, RandomizationConfigResponse.class);
+    public void setFlagsFromJsonString(String jsonString) {
+        RandomizationConfigResponse config = gson.fromJson(jsonString, RandomizationConfigResponse.class);
```

Comment:
This now takes and passes to Gson a string instead of a reader.

This should be fine, provided our responses are relatively (<100mb) small, which they will be.